cf-java-logging-support-servlet/pom.xml

```diff
@@ -76,4 +76,20 @@
 		</dependency>
 
 	</dependencies>
+
+	<build>
+		<plugins>
+			<plugin>
+				<artifactId>maven-source-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>jar</goal>
+							<goal>test-jar</goal>
```

Comment:
Will this created more artefacts, that will be uploaded to Maven Central. Note, that in the parent pom.xml, there is the "ossrh" profile, that creates, signs and uploads the artefacts. This already contains a `maven-source-plugin`. Do you have an idea, how these two configurations interact?

Reply:
You could be right, that could really a problem. If you have look into the target of this module, it contains 2 additional jars. Can somehow control which artifacts are uploaded to Maven central? Do you have something like an artifact or module list? To be hones I'm not familiar with this upload.

Reply:
I tested the publishing. It almost looks fine, but for the jar, that contains the sources. It does not exists, since its execution phase is so early in the Maven lifecycle, that all the actions for providing the correct code come afterwards. I will add the required configuration, once this PR is merged. Without a sources jar, Maven Central would reject this project.